x-pack/plugins/security/public/views/management/edit_role/lib/constants.js

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const CLUSTER_PRIVS_DOC_LINK = `https://www.elastic.co/guide/en/x-pack/current/security-privileges.html#security-privileges`;
```

Comment:
Can we move this to https://github.com/elastic/kibana/blob/master/src/ui/public/documentation_links/documentation_links.js which we use to build other doc links?

Reply:
TIL - will do!